### ISSUE TITLE
Open3D-ML supported on Windows 10 ? 

### ISSUE THREAD

**Issue by maohong30** [intel-isl/Open3D-ML#303]
Hi. Is the latest Open3D with ML module supported on Windows ? 

When building from source using Cmake,

> DBUILD_TENSORFLOW_OPS=ON
> DBUNDLE_OPEN3D_ML=ON

I receive the error below:

> CMake Error at cpp/open3d/ml/CMakeLists.txt:2 (message):
>   Building TensorFlow ops on Windows is currently not supported.

From an issue #1973 June 2020 last year, I realize Open3D-ML is not supported on Windows.
The workaround suggested was removing line 2 of file Open3D/cpp/open3d/ml/CMakeLists.txt
> Line 2 : message( FATAL_ERROR "Building Tensorflow ops on Windows is currently not supported." ) 

 
https://github.com/intel-isl/Open3D/issues/1973#issuecomment-646212684

If Open3D-ML is not supported, is there any other workaround I can try?
Thank You.

**Environment :**

Operating system: Windows 10 64-bit
Python version: Python 3.8
Open3D version: 0.13
Is this remote workstation?: no
How did you install Open3D?: build from source
Compiler version (if built from source): Visual studio 2019 Win 64
Cuda: 10.1
Graphic Card: RTX2070

**Comment by hduonggithub** [comment]
@maohong30 

Not support for Windows at the moment!

**Comment by maohong30** [comment]
Does anyone know if the future release of Open3D-ML will support Windows? 

I am really impressed with the great job Open3D has done for a beginner like me in the field of 3D Data Processing. While I'm also working on ML, I really hope the ML extension will also be available for Windows users. 

Thank you very much

**Comment by sanskar107** [comment]
Hi @maohong30 
We disabled building the tensorflow ops on Windows because we ran into some issues, e.g. tensorflow 2.0 did not ship with the required libs and headers for building ops. We did not look into this again with a more recent verison of tensorflow. If you want to give it a try, you can simply remove remove this line 
https://github.com/intel-isl/Open3D/blob/master/cpp/open3d/ml/CMakeLists.txt#L2

Anyway, Windows WSL can be considered as a workaround at the moment.

**Comment by maohong30** [comment]
> Hi @maohong30
> We disabled building the tensorflow ops on Windows because we ran into some issues, e.g. tensorflow 2.0 did not ship with the required libs and headers for building ops. We did not look into this again with a more recent verison of tensorflow. If you want to give it a try, you can simply remove remove this line
> https://github.com/intel-isl/Open3D/blob/master/cpp/open3d/ml/CMakeLists.txt#L2
> 
> Anyway, Windows WSL can be considered as a workaround at the moment.

Thank you very much.

**Comment by hduonggithub** [comment]
I confirm I did use WSL 2 for Open3D-ML. Everything (+ GPU based for both tensorflow and torch) is good so far (but I did not try the latest version 0.13)

Thanks @sanskar107 for this info. I have a minor question for you. I have modified the vis_gui.py in Open3D for my gui. My gui now can run under WSL2 too. My issue now is that this gui does not run/utilize GPU and as a result, the gui becomes very slow to zoom/pan/rotate when adding point cloud. I do not have this issue in Windows where I can assign what python.exe to be used with GPU in Windows. Any idea? Thanks!
